lightning_hpo/controllers/sweep.py

```diff
@@ -156,6 +156,22 @@ def stop_experiment(self, config: StopExperimentConfig):
                     return f"The current experiment `{experiment.name}` has been stopped."
         return f"The current experiment `{config.name}` doesn't exist."
 
+    def show_logs(self) -> List[ShowLogsConfig]:
+        data = []
+        for sweep in self.db.select_all():
+            if sweep.sweep_id in self.r:
+                works = self.r[sweep.sweep_id].works()
+                if sweep.algorithm:
+                    data.append(ShowLogsConfig(name=sweep.sweep_id, components=[w.name for w in works]))
```

Comment:
Wouldn't this get tensorboard as well?

Reply:
No, I didn't add support for tensorboard as this wasn't within provided API. Right now, supporting only sweeps and experiments.

Reply:
That's totally fine, I definitely wasn't looking for tensorboard logs :-) 
My question was: when querying the works, wouldn't tensorboard come up when using the experiment name? (I can't recall the naming of tensorboard works)

Reply:
No, it isn't within another machine.